lib/membrane_h264_plugin/parser.ex

```diff
@@ -66,18 +90,18 @@ defmodule Membrane.H264.Parser do
       %H264{alignment: alignment, nalu_in_metadata?: true} when alignment in [:nalu, :au]
 
   def_options sps: [
-                spec: binary(),
-                default: <<>>,
+                spec: binary() | [binary()],
+                default: [],
                 description: """
-                Sequence Parameter Set NAL unit binary payload - if absent in the stream, should
-                be provided via this option.
+                Sequence Parameter Set NAL unit binary payloads - if absent in the stream, should
+                be provided via this option (only available for `:annexb` output stream structure).
                 """
               ],
               pps: [
-                spec: binary(),
-                default: <<>>,
+                spec: binary() | [binary()],
+                default: [],
                 description: """
-                Picture Parameter Set NAL unit binary payload - if absent in the stream, should
+                Picture Parameter Set NAL unit binary payloads - if absent in the stream, should
                 be provided via this option.
```

Comment:
isn't it the same case as for `sps` option? (that it's only available for `:annexb` output stream structure)